docs/src/pages/components/buttons/ColorButtons.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+
+export default function ColorButtons() {
+  return (
+    <Box
+      sx={{
+        '& > :not(style)': { m: 1 },
+      }}
```

Comment:
For all cases it might be better to leave them in a single line.

```suggestion
      sx={{ '& > :not(style)': { m: 1 } }}
```